v3/plugins/redhat/java8/0.55.1/meta.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: v2
+publisher: redhat
+name: java8
+version: 0.55.1
+type: VS Code extension
+displayName: Language Support for Java 8
+title: Language Support for Java(TM) by Red Hat
+description: Java Linting, Intellisense, formatting, refactoring, Maven/Gradle support and more...
+icon: https://www.eclipse.org/che/images/logo-eclipseche.svg
+repository: https://github.com/redhat-developer/vscode-java
+category: Language
+firstPublicationDate: "2020-02-05"
+spec:
+  containers:
+    - image: "quay.io/eclipse/che-sidecar-java:8-0cfbacb"
+      name: vscode-java
+      memoryLimit: "1500Mi"
+      volumes:
+      - mountPath: "/home/theia/.m2"
+        name: m2
+  extensions:
+    - https://download.jboss.org/jbosstools/vscode/3rdparty/vscode-java-debug/vscode-java-debug-0.24.0.vsix
+    - https://download.jboss.org/jbosstools/static/jdt.ls/stable/java-0.55.1-1984.vsix
+    
```

Comment:
Please add a trailing newline in case some of the build scripts expect it.

Reply:
Requiring an _empty_ newline at the end of the file is so stupid it makes me want to scream. But yeah, fair point: I'll bow to the great machine

Reply:
https://i.redd.it/zibdmw9m66141.jpg You're not the only one annoyed by syntactic whitespace :)